### PR TITLE
Migrate messages to proto

### PR DESCRIPTION
**Pull request by sahith-narahari**
This PR migrates all 3 msgs of wasm to proto definitions
- Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/master/CONTRIBUTING.md#pr-targeting))

- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote tests
- [ ] Updated relevant documentation (`docs/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ This code was forked from the `cosmos/gaia` repository and the majority of the c
 
 **Note**: Requires [Go 1.14+](https://golang.org/dl/)
 
-**Compatibility**: Last merge from `cosmos/gaia` was `3f4572d3d74c99584f44eb4a0ee06bd340d48867` (Apr 11, 2020)
+**Compatibility**: Last merge from `cosmos/gaia` was `20a071cf84b965d0892c5a216b32c44963a83173` (Apr 27, 2020)
```
Comment (ethanfrey) [inline README.md:16]: :+1:

File: x/wasm/client/cli/query.go [hunk 2]
```diff
@@ -19,8 +19,8 @@ import (
 	"github.com/cosmos/cosmos-sdk/codec"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
-	"github.com/cosmwasm/wasmd/x/wasm/internal/keeper"
```
Comment (ethanfrey) [inline x/wasm/client/cli/query.go:22]: They decided to change the "standard module structure" again?

File: x/wasm/types/types.proto [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+message MsgInstantiateContract{
+    option (gogoproto.goproto_getters) = false;
+
+    bytes sender = 1 [(gogoproto.casttype) = "github.com/cosmos/cosmos-sdk/types.AccAddress"];
+    uint64 code = 2 [(gogoproto.moretags) = "yaml:\"code_id\""];
+    string label = 3;
+    bytes init_msg = 4 [(gogoproto.casttype) = "encoding/json.RawMessage"];
+    repeated cosmos_sdk.v1.Coin init_funds = 5 [
```
Comment (ethanfrey) [inline x/wasm/types/types.proto:28]: Where is this imported from by the way?

Is there a good link on how to compile this (which tooling)? (So I can modify the proto if needed)

File: x/wasm/client/cli/query.go (hunk 2 above)
Comment (sahith-narahari) [reply x/wasm/client/cli/query.go:22]: sadly, yes. There's no differentiation of internal and external anymore

File: x/wasm/types/types.proto (hunk 3 above)
Comment (sahith-narahari) [reply x/wasm/types/types.proto:28]: you need to run make proto-gen, so that it generates types.pb.go

**Pull request merged**